HeaderView.java

```diff
@@ -0,0 +1,48 @@
+package com.example.vkpage;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+public class HeaderView extends LinearLayout {
+
+    private ImageView userImage;
+    private TextView userName;
+    private TextView userEmail;
+
+    public HeaderView(Context context) {
+        super(context);
+
+        init();

```

Comment:
Instead of call init in all constructors, you can write it like this 

`public HeaderView(Context context) {
 this(context, null);
}`
`public HeaderView(Context context, @Nullable AttributeSet attributeSet) {
 this(context, null, 0);
}`
`public HeaderView(Context context, @Nullable AttributeSet attributeSet, int defStyleAttribute) {
 super(context, attributeSet, defStyleAttribute);
 init();
}`